### DISCUSSION TITLE
Welcome to ChatGPT-Comments-Reply-WordPress-Plugin Discussions!

### DISCUSSION THREAD

**Discussion by VolkanSah** [VolkanSah/aicc-comments-reply#1] [Announcements]
## 👋 Welcome to the ChatGPT Comments Reply Plugin Discussion!

We're using Discussions as a place for our community to connect, share ideas, and get support related to the ChatGPT Comments Reply Plugin. We hope that you:

* Ask questions about the plugin, its features, or its implementation.
* Share your experiences, ideas, or customizations for the plugin.
* Engage with other community members and help each other out.
* Welcome newcomers and maintain an open-minded, collaborative environment.

To get started, introduce yourself by commenting below, and let us know how you're using the ChatGPT Comments Reply Plugin in your WordPress site.

📢 **Announcement**: We're excited to launch the Discussions feature for our community! Join the conversation and share your thoughts, questions, and ideas related to the ChatGPT Comments Reply Plugin.

🔗 For any issues related to the plugin, please use the [Issues tab](https://github.com/VolkanSah/ChatGPT-Comments-Reply-WordPress-Plugin/issues) to report them. This will help us keep the Discussions focused on ideas, questions, and community engagement.

➡️ Feel free to create new discussion threads for specific topics related to the plugin, such as feature requests, troubleshooting, or sharing your customizations.

Let's build a thriving community around the ChatGPT Comments Reply Plugin together! 💪

**Comment by wpfyorg** [comment]
Hey! 

Is it possible to send the content of post as context to generate reply?

**Comment by VolkanSah** [comment]
This is done like this

**Reply by wpfyorg** [reply]
Sorry, I didn't get you.
